src/main/java/org/spongepowered/common/bridge/world/storage/MapStorageBridge.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * This file is part of Sponge, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.common.bridge.world.storage;
+
+import net.minecraft.world.storage.MapData;
+
+import java.util.Optional;
+
+public interface MapStorageBridge {
+    Optional<MapData> bridge$getMinecraftMapData(int id);
+
+    /**
+     * Only call if you know what you are doing, if used
+     * incorrectly, will cause maps to be overwritten.
+     * Next map created will use this id + 1
+     * @param id int To set highest map number to.
+     */
+    void bridge$setHighestMapId(short id);
```

Comment:
getHighestMapId, gets the highest map id, that exists, so when 1 map exists, its 0, etc. It doesn't make much sense to return -1 or something, because if you actually tried to get that map id, it wouldn't exist.
I will update the javadoc to make this more clear.